product_supplierinfo_for_customer/models/product_customerinfo.py

```diff
@@ -11,6 +11,15 @@ class ProductCustomerInfo(models.Model):
 
     name = fields.Many2one(string="Customer", help="Customer of this product")
 
+    def onchange(self, values, field_name, field_onchange):
+        if "product_tmpl_id" in values and field_name != "product_tmpl_id":
```

Comment:
Is this a problem really linked to this module behaviour or a more general problem ? Otherwise, this should be elsewhere

Reply:
@rousseldenis Yes this is more general. I've created issue here https://github.com/odoo/odoo/issues/87192

Reply:
@rousseldenis any hint on what we can do to get the issue fixed on odoo? feel quite helpless about this 🤷‍♂️